VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: Inverters_Front Inverters_Rear


BO_ 177 AP_ETAS_EnableInv: 8 Vector__XXX
 SG_ Enable_Inv_R : 0|1@0+ (1,0) [0|1] "-"  Inverters_Rear

BO_ 180 TBD_Inv_R_Power: 6 Inverters_Rear
 SG_ TotalPower : 7|16@0+ (1,0) [0|65535] "kW" Vector__XXX
 SG_ Power_RR : 39|16@0+ (1,0) [0|65535] "kW" Vector__XXX
 SG_ Power_RL : 23|16@0+ (1,0) [0|65535] "kW" Vector__XXX

BO_ 199 TBD_Inv_R_Torque: 4 Inverters_Rear
 SG_ TorqueReal_RR : 23|16@0+ (1,0) [0|65535] "Nm" Vector__XXX
 SG_ TorqueReal_RL : 7|16@0+ (1,0) [0|65535] "Nm" Vector__XXX

BO_ 176 AP_Inv_R_Error: 8 Inverters_Rear

BO_ 181 AP_Inv_R_Speed: 4 Inverters_Rear
 SG_ SpeedMotor_RR : 23|16@0+ (1,0) [0|65535] "rpm" Vector__XXX
 SG_ SpeedMotor_RL : 7|16@0+ (1,0) [0|65535] "rpm" Vector__XXX

BO_ 179 AP_Inv_R_DCBus: 4 Inverters_Rear
 SG_ DCBus_RR : 23|16@0+ (1,0) [0|65535] "V" Vector__XXX
 SG_ DCBus_RL : 7|16@0+ (1,0) [0|65535] "V" Vector__XXX

BO_ 201 MEAS_Inv_R_Currents_R: 6 Inverters_Rear
 SG_ Iq_RR : 39|16@0+ (1,0) [0|65535] "A" Vector__XXX
 SG_ Icmd_RR : 23|16@0+ (1,0) [0|65535] "A" Vector__XXX
 SG_ Iactual_RR : 7|16@0+ (1,0) [0|65535] "A" Vector__XXX

BO_ 200 MEAS_Inv_R_Currents_L: 6 Inverters_Rear
 SG_ Iq_RL : 39|16@0+ (1,0) [0|65535] "A" Vector__XXX
 SG_ Icmd_RL : 23|16@0+ (1,0) [0|65535] "A" Vector__XXX
 SG_ Iactual_RL : 7|16@0+ (1,0) [0|65535] "A" Vector__XXX

BO_ 178 AP_Inv_R_Temperatures: 8 Inverters_Rear
 SG_ TempMotor_RR : 55|16@0+ (1,0) [0|65535] "ºC" Vector__XXX
 SG_ TempMotor_RL : 39|16@0+ (1,0) [0|65535] "ºC" Vector__XXX
 SG_ TempIGBT_RR : 23|16@0+ (1,0) [0|65535] "ºC" Vector__XXX
 SG_ TempIGBT_RL : 7|16@0+ (1,0) [0|65535] "ºC" Vector__XXX



BA_DEF_  "MultiplexExtEnabled" ENUM  "No","Yes";
BA_DEF_  "BusType" STRING ;
BA_DEF_ BU_  "NodeLayerModules" STRING ;
BA_DEF_ BU_  "ECU" STRING ;
BA_DEF_ BU_  "CANoeJitterMax" INT 0 0;
BA_DEF_ BU_  "CANoeJitterMin" INT 0 0;
BA_DEF_ BU_  "CANoeDrift" INT 0 0;
BA_DEF_ BU_  "CANoeStartDelay" INT 0 0;
BA_DEF_DEF_  "MultiplexExtEnabled" "No";
BA_DEF_DEF_  "BusType" "";
BA_DEF_DEF_  "NodeLayerModules" "";
BA_DEF_DEF_  "ECU" "";
BA_DEF_DEF_  "CANoeJitterMax" 0;
BA_DEF_DEF_  "CANoeJitterMin" 0;
BA_DEF_DEF_  "CANoeDrift" 0;
BA_DEF_DEF_  "CANoeStartDelay" 0;

